tests/unit/db/account_tests.py

```diff
@@ -0,0 +1,600 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+_account_tests_
+
+account module - Unit tests for CouchDB and Cloudant account classes
+
+"""
+
+import unittest
+import requests
+import json
+import base64
+from datetime import datetime
+
+from cloudant.account import CouchDB, Cloudant
+from cloudant.errors import CloudantException
+
+# Set RUN_CLOUDANT_TESTS to True if you wish to execute the Cloudant 
+# class tests.  Defaults to False in order to skip Cloudant tests in Travis-CI.
+RUN_CLOUDANT_TESTS = False
```

Comment:
These need a way of being set from command line or environment variables. Ideally they should be similar to the ones used for other libraries like java-cloudant and sync-android.